importer/Uploader.py

```diff
@@ -246,17 +245,18 @@ def upload(self):
     def set_wd_item(self):
         item_exists = True if self.data["wd-item"] is not None else False
         if item_exists:
-            # item_q = self.data["wd-item"]
-            self.wd_item = self.wdstuff.QtoItemPage(self.TEST_ITEM)
-            print(self.wd_item)
+            item_q = self.data["wd-item"]
```

Comment:
I would probably keep the test item options. Maybe something like:
```
if self.test:
    self.wd_item = self.wdstuff.QtoItemPage(self.TEST_ITEM)
elif item_exists:
    item_q = self.data["wd-item"]
    self.wd_item = self.wdstuff.QtoItemPage(item_q)
else:
    self.wd_item = self.create_new_item(self.log)
```
That way you can toggle testing on/off (which is useful for later datasets) without having to update deep parts of the code. You could even set it as a command line argument.